modules/mobile/app/models/mobile/v0/adapters/vaos_v2_appointment.rb

```diff
@@ -345,19 +343,23 @@ def parse_phone(phone)
           # and optional extension (until the end of the string) (?:\sx(\d*))?$
           phone_captures = phone&.match(/^\(?(\d{3})\)?.?(\d{3})-?(\d{4})(?:\sx(\d*))?$/)
 
```

Comment:
This error was happening, but I believe it was only happening because the phone number was nil. I've changed the logic to only log if the phone number is present but phone captures are nil, which would imply that our regex capture failed.